cantools/database/can/formats/dbc.py

```diff
@@ -1386,8 +1386,11 @@ def get_send_type(frame_id_dbc):
 
         try:
             result = message_attributes['GenMsgSendType'].value
-            # Resolve ENUM index to ENUM text
-            result = definitions['GenMsgSendType'].choices[int(result)]
+            
+            # if definitions is enum (otherwise above value is maintained) -> Prevents ValueError
+            if definitions['GenMsgSendType'].choices != None:
```

Comment:
I'm not 100% sure, but this seems to be more bullet proof as it (probably) works if `definitions['GenMsgSendType'].choices` is defined but does not exhibit an entry for `result`?
```suggestion
            if definitions['GenMsgSendType'].choices is not None and \

                int(result) in definitions['GenMsgSendType'].choices:
```